src/bidiMapper/modules/browser/BrowserProcessor.ts

```diff
@@ -113,7 +113,7 @@ export class BrowserProcessor {
       {targetId},
     );
     return {
-      // Is not supported in CDP yet.
+      // `activate` is not supported in CDP yet.
```

Comment:
```suggestion
      // `active` is not supported in CDP yet.
```

FWIW I would default to true. In general, I am not sure how useful this data will be to WebDriver BiDi client given that the input can be sent to any browsing context.